src/__tests__/index.test.ts

```diff
@@ -385,6 +385,13 @@ const vectors: Array<TestVector> = [
       },
     ],
   },
+  {
+    name: 'SIA',
+    coinType: 1991,
+    passingVectors: [
+      { text: '0x314159265dD8dbb310642f98f50C066173C1259b', hex: '314159265dd8dbb310642f98f50c066173c1259b' },
```

Comment:
Where did you get this test data? Your reference https://sia.tech/docs/#wallet-address-get seems quite different and there seems no checksum.